deploy/charts/rook-ceph-cluster/values.yaml

```diff
@@ -643,45 +643,46 @@ cephObjectStores:
 
 # cephECStorageClass also is disabled by default, please remove the comments and set desired values to enable it
 # if cephECBlockPools are uncommented you must remove the comments of cephEcStorageClass as well
-#cephECStorageClass:
-#  name: rook-ceph-block
-#  # Change "rook-ceph" provisioner prefix to match the operator namespace if needed
-#  provisioner: rook-ceph.rbd.csi.ceph.com # driver:namespace:operator
-#  parameters:
-#    # clusterID is the namespace where the rook cluster is running
-#    # If you change this namespace, also change the namespace below where the secret namespaces are defined
-#    clusterID: rook-ceph # namespace:cluster
+#cephECStorageClasses:
```

Comment:
this file only includes space changes right?

Reply:
It's now a list, so line 647 now has a `-` for the list.

Reply:
yes @subhamkrai  just adjust the yaml to support the list now